common/lm9ds1_imu.h

```diff
@@ -766,6 +767,8 @@ class LM9DS1
                 return 0.0175f;
             case GyroScale::DPS2000:
                 return 0.07f;
+            default:
```

Comment:
As these are modern enums, surely this isn't necessary? Also if it is - it's definitely an error so should probably throw rather than return anything that could be ignored/cause maths horror

Reply:
I'd assume not, but gcc was giving me warnings (I think now that we have -Wpedantic enabled). It is a little odd. I suppose maybe theoretically one can cast a ``uint8_t`` to ``GyroScale`` even if it's not a valid value?

I'll change it to a throw instead.

Reply:
Ahh ok - throw in that case! I feel newer gccs are getting _more_ pedantic - on Ubuntu 18 it raises some new warnings deep in a particularly agregious bit of GeNN